staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -21,6 +21,18 @@ import (
 	"net/http"
 	"time"
 
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
```

Comment:
I won't ask for a change but making large reorganizations to import lists doesn't make this PR any smaller :)

Reply:
But ... there was an extra empty line that shouldn't be there? :-)